### PR TITLE
Instruction file merged

### PR DESCRIPTION
**Pull request by ritwiksharma049**
I have merged my installation file with the existing one. Please have a look once at the file. 

Thank you

### CODE REVIEW

File: doc/installation.md [hunk 1]
```diff
@@ -1,85 +1,158 @@
...
 
-Another possibility is to do it all in one go with:
 
-	conda install -c conda-forge parallel_bilby
+After forking, run the following command to clone the repository into your currently directory (by default, in your home directory):
 
-although that command hangs on certain systems.
 
-The next hurdle is pymultinest, and again, we suggest using conda
+* $ git clone https://github.com/ritwiksharma049/nmma   (replace ritwiksharma049 with your GitHub username)
```
Comment (mcoughlin) [inline doc/installation.md:41]: I suggest replacing your username with your_username_here or similar.

File: doc/installation.md [hunk 2]
```diff
@@ -101,3 +174,15 @@ After removing the matplotlib cache:
 	rm ~/.cache/matplotlib -rf
 
 Beautiful fonts should be yours.
+
```
Comment (mcoughlin) [inline doc/installation.md:177]: No need for spaces right?

File: doc/installation.md [hunk 3]
```diff
@@ -1,85 +1,158 @@
-## Installation
+# Instructions For NMMA Installation
 
-### anaconda3
+## Preliminary Steps:
 
-First of all, we suggest you install [anaconda](https://docs.anaconda.com/anaconda/install/linux/), in order to avoid conflicts linked to different versions of pythons or pip. Afterwards, update conda:
```
[the remark this answers is not in the document]
Comment (ritwiksharma049) [reply doc/installation.md:5]: I have mentioned that the installation procedure is primarily for linux systems and windows users are advised to use WSL. Then the subsequent commands are common for linux terminal and WSL.

File: doc/installation.md [hunk 4]
```diff
@@ -1,85 +1,158 @@
...
+Ubuntu 20.04 is available for free download on Microsoft Store. 
 
-	conda update --all
+**Installing Anaconda3**
 
+On your Linux/WSL terminal, run the following commands to install anaconda (replace 5.3.1 by the latest version):
 
-### Cloning nmma
 
-Then, you should go ahead and git clone the repository. Note that we link below to the main branch, but suggest making changes on your own fork (please also see our [contributing guide](./contributing.html)).
```
[the remark this answers is not in the document]
Comment (ritwiksharma049) [reply doc/installation.md:12]: Will add.

File: doc/installation.md (hunk 1 above)
Comment (ritwiksharma049) [reply doc/installation.md:41]: Sure.

**Review by mcoughlin — APPROVED** [review]

**Comment by mcoughlin** [comment]
Thanks @ritwiksharma049 we can continue working on this as we add other things.

**Comment by ritwiksharma049** [comment]
Yes, this file should solve some major problems, if not all.

**Pull request merged**